### ISSUE TITLE
Documentation is heavily out of date.

### ISSUE THREAD

**Issue by zkldi** [TNG-dev/tachi-docs#20]
Tachi-Server has been progressing at a blistering rate, but that means that the documentation is missing bits. As part of the feature freeze, we will need to go over all the documentation and update the bits that are out of date. Fun!

**Comment by pfych** [comment]
https://tachi.readthedocs.io/en/latest/tachi-server/setup/config/

Setup config is missing documentation on the `TACHI_CONFIG` object

**Comment by zkldi** [comment]
Yup, looks like conf json5 is incredibly outdated. I'll get around that now.

**Comment by zkldi** [comment]
Handled.

**Comment by zkldi** [comment]
There's no documentation for `import/poll-status` atm, need to sort that out.
